pkg/restic/config_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package restic
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	arkv1api "github.com/heptio/ark/pkg/apis/ark/v1"
+)
+
+func TestGetRepoIdentifier(t *testing.T) {
+	// TODO decide how to mock out aws.GetBucketRegion
```

Comment:
interested in input here -- this is a good example of the general question "how to unit-test package-level functions that have dependencies?" I could create an interface/struct, `BucketRegionGetter`, and require that as an arg to the function - but that obviously makes the signatures more complicated and requires more object construction. Any other good ideas?

Reply:
@heptio/bluesteelsquad for input. I can't think of anything other than putting it behind an interface/struct offhand.

Reply:
See latest commit for what it looks like